app/src/main/java/org/literacyapp/authentication/helper/DetectionHelper.java

```diff
@@ -154,6 +153,20 @@ private static synchronized void setScreenBrightness(Context context, double cur
         }
     }
 
+    private static synchronized void setDisplayTemperatureNight(){
+        try {
+            int displayTemperatureNight = getDisplayTemperatureNight();
+            if (displayTemperatureNight <= DISPLAY_TEMPERATURE_NIGHT_DEFAULT){
```

Comment:
I don't understand this check. If the current temperature is not larger than 2500K, it's adjusted to 4500K? Shouldn't it be `if (displayTemperatureNight <= DISPLAY_TEMPERATURE_NIGHT_BRIGHTER)`?